configs/Lightning-AI/metrics_torch-develop.yaml

```diff
@@ -0,0 +1,38 @@
+target_repository:
+  HTTPS: https://github.com/Lightning-AI/metrics.git
+  # checkout a particular branch or a tag
+  checkout: master
+  # define installing package extras
+  install_extras:
+    - all
+    - test
+  # install additional requirements from a file
+  requirements_file: requirements/audio_test.txt
+  # copy some  test from the target repository
+  copy_tests:
+    - tests/unittests
+
+contact:
+  slack:
+    # Nicki Skafte
+    - USZ6EQ8A2
```

Comment:
@justusschock should we have you here too? 🐶

Reply:
```suggestion
contact:
  slack:
    # Nicki Skafte
    - USZ6EQ8A2
    # Justus Schock
    - U010XFPFETH
```